navigation/navigation-runtime/src/main/java/androidx/navigation/NavController.kt

```diff
@@ -496,8 +496,8 @@ public open class NavController(
     /**
      * Attempts to pop the controller's back stack back to a specific destination.
      *
-     * @param route The topmost destination to retain. May contain filled in arguments as long as
-     * it is exact match with route used to navigate.
+     * @param route The route of the topmost destination to retain. Both route patterns
+     * and route instances with filled in arguments are supported.
```

Comment:
The use of "route pattern" and "route" here should be distinctive enough. Can you also clarify that if using route, it must match the original route that was used to navigate?

Reply:
Agree on "route pattern" and "route" 👍 

I removed the `May contain filled in arguments as long as it is exact match with route used to navigate.` because I felt like this was the general expectation of the method and didn't add much information. Whether using a route pattern or a route, the expectation is that it returns true if found or false if not. Or did I get this wrong?

Reply:
`returns true if found or false if not` is indeed the case. In the past we matched based on route pattern and didn't differentiate between "route/myArg1" and "route/myArg2". Now that matching takes into account filled in arguments, matching behavior has changed and what was previously expected to match may no longer match. Therefore the clarification could be important. Also, it doesn't hurt to have more docs than less :).